src/status_im2/contexts/onboarding/common/carousel/animation.cljs

```diff
@@ -16,28 +17,48 @@
     (js-obj "duration" (or duration progress-bar-animation-duration)
             "easing"   (:linear reanimated/easings)))))
 
+(defn calculate-remainder
+  [dividend divisor]
+  (if (and (not= 0 dividend) (zero? (mod dividend divisor)))
+    divisor
+    (mod dividend divisor)))
+
+(defn animation-value
+  [q index]
+  (let [next-quotient (+ q index)]
+    (calculate-remainder (* (if (>= next-quotient initial-division-quotient)
```

Comment:
I feel like this is easier to read if you use a named variable and then also move the `if` outside the multiplication. For something so simple it might be easier to just have it as:

```
(defn animation-value
  [q index]
  (let [next-quotient (+ q index)
        dividend (if (>= next-quotient initial-division-quotient)
                   (* (- next-quotient initial-division-quotient) 25)
                   (* next-quotient 25))]
    (calculate-remainder dividend 100)))
    
   ```

Reply:
also consider putting `25` and `100` in a named def in this file to help explain their use case here 👍